src/pivotal-ui/components/whitespace/whitespace.scss

```diff
@@ -59,7 +66,7 @@ See [list spacing][list_whitespace].
 @include whitespace(($whitespace-s, $whitespace-m, $whitespace-l, $whitespace-xl, $whitespace-xxl, $whitespace-xxxl, $whitespace-xxxxl));
 
 @media all and (max-width: $screen-sm) {
-  @include whitespace(($whitespace-screen-sm-s, $whitespace-screen-sm-m, $whitespace-screen-sm-l, $whitespace-screen-sm-xl, $whitespace-screen-sm-xxl, $whitespace-screen-sm-xxxl, $whitespace-screen-sm-xxxxl));
+  @include whitespace(($whitespace-screen-sm-s, $whitespace-screen-sm-m, $whitespace-screen-sm-l, $whitespace-screen-sm-xl, $whitespace-screen-sm-xxl, $whitespace-screen-sm-xxxl, $whitespace-screen-sm-xxxxl), 4, (p,m));
```

Comment:
In the defaults at https://github.com/npm/pivotal-ui/pull/134#discussion-diff-60452149R66